skorch/tests/callbacks/test_lr_scheduler.py

```diff
@@ -35,6 +35,41 @@ def test_simulate_lrs_batch_step(self, policy):
         expected = np.array([1, 2, 3, 4, 5, 4, 3, 2, 1, 2, 3])
         assert np.allclose(expected, lrs)
 
+    def test_simulate_lrs_reduced_lr_on_plateau_scalar(self):
+        # Feed a constant, scalar "loss" to the scheduler.
+        lr_sch = LRScheduler(
+            ReduceLROnPlateau, factor=0.1, patience=1,
+        )
+        lrs = lr_sch.simulate(
+            steps=5, initial_lr=1, step_args=0.5
+        )
+        # O = OK epoch
+        # I = intertolerable epoch
```

Comment:
What is "intertolerable"?

Reply:
This is terminology taken from the [docs](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ReduceLROnPlateau.html#torch.optim.lr_scheduler.ReduceLROnPlateau)

> In the second epoch, if the performance is worse than the baseline, we have what is considered an intolerable epoch.

Drastic terminology but I wanted to adhere to already defined terms :)

Reply:
Drastic indeed, but it's in the Shakespeare corpus, so it must be a common English word.